Source/Fuse.Nodes/Visual.Children.uno

```diff
@@ -29,24 +29,24 @@ namespace Fuse
 
 		public T FirstChild<T>() where T: Node
 		{
-			for (var i = 0; i < Children.Count; ++i)
+			var c = Children_first;
+			while (c != null)
 			{
-				var c = Children[i] as T;
-				if (c != null)
-					return c;
+				var v = c as T;
+				if (v != null) return v;
+				c = c.Children_next;
 			}
 			return null;
 		}
 
-		public void RemoveAllChildren<T>()
+		public void RemoveAllChildren<T>() where T: Node
```

Comment:
This restriction really isn't needed, `where T : class` is enough. Somebody (not us though) might be calling this with an interface that doesn't derive from `Node`.